compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3093,7 +3186,7 @@ TR::Register *OMR::Power::TreeEvaluator::inlineVectorBinaryOp(TR::Node *node, TR
 
    if (masked)
       {
-      TR_ASSERT_FATAL(resReg->getKind() == TR_VSX_VECTOR, "Only masked VSX opcodes are currenlty supported\n");
+      TR_ASSERT_FATAL(resReg->getKind() == TR_VSX_VECTOR || resReg->getKind() == TR_VRF, "Only masked VSX opcodes are currenlty supported\n");
```

Comment:
If we add TR_VRF, the text of the assertion does not make sense. I will try to recall why it was added in the first place.

Reply:
I think it was my mistake to add this ASSERT in some earlier PR and it can be removed. But a comment "Note: both VRF and VSX registers are supported by xxsel" would be helpful I think.